modules/ROOT/pages/deploy-to-runtime-fabric.adoc

```diff
@@ -10,11 +10,12 @@ Deploy a Mule application to Anypoint Runtime Fabric and test the inbound traffi
 Before deploying an application to Anypoint Runtime Fabric, you must perform the following:
 
 * Install and configure Anypoint Runtime Fabric. See xref:install-create-rtf-arm.adoc[Begin Runtime Fabric Installation] for more information.
-* Ensure that Runtime Fabric is associated with an environment.
+* Ensure that Runtime Fabric is associated with an environment. See xref:associate-environments.adoc[Associate Business Groups and Environments to Anypoint Runtime Fabric]
 * Determine the resource allocation requirements for your application. See xref:deploy-resource-allocation.adoc[Determine Application Resource Allocation on Runtime Fabric] for more information.
 * Your Anypoint user may require the *Exchange Contributors* permission to deploy applications to Runtime Fabric.
+* To deploy an application to a specific domain (endpoint), ensure that you have setup domains (endpoints) in your environment. See xref:enable-inbound-traffic-self.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric (Self-managed Kuberenetes)] or xref:enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric (VM / Baremetal)]
```

Comment:
@sundippatel @sarjeet2013 We use "domain" when deploying an app, but "endpoint" when enabling inbound traffic. Is there a conceptual/technical difference? Or are they referring to the same thing?

Reply:
Also, Shayan mentioned that we prefer that users use domains. Let me know if we need to call this out more.

Reply:
After my with Rohit, I think this is resolved.